matter_server/server/device_controller.py

```diff
@@ -90,11 +98,23 @@
 DESCRIPTOR_PARTS_LIST_ATTRIBUTE_PATH = create_attribute_path_from_attribute(
     0, Clusters.Descriptor.Attributes.PartsList
 )
+BASIC_INFORMATION_VENDOR_ID_ATTRIBUTE_PATH = create_attribute_path_from_attribute(
+    0, Clusters.BasicInformation.Attributes.VendorID
+)
+BASIC_INFORMATION_PRODUCT_ID_ATTRIBUTE_PATH = create_attribute_path_from_attribute(
+    0, Clusters.BasicInformation.Attributes.ProductID
+)
 BASIC_INFORMATION_SOFTWARE_VERSION_ATTRIBUTE_PATH = (
     create_attribute_path_from_attribute(
         0, Clusters.BasicInformation.Attributes.SoftwareVersion
     )
 )
+BASIC_INFORMATION_SOFTWARE_VERSION_STRING_ATTRIBUTE_PATH = (
+    create_attribute_path_from_attribute(
+        0, Clusters.BasicInformation.Attributes.SoftwareVersionString
+    )
+)
+
```

Comment:
In a follow-up iteration we should maybe start moving all these constants into the constants module

Reply:
well, hmm maybe not as these are not plain/simple constants but created from a util function.

Reply:
We probably could add a `to_attribute_path(endpointId: int)` function to the `ClusterAttributeDescriptor`. With this, we could do:

```python
node.attributes.get(str(Clusters.BasicInformation.Attributes.SoftwareVersionString.to_attribute_path(0)))
```